osu.Game/Overlays/Direct/DirectPanel.cs

```diff
@@ -87,5 +260,30 @@ public Statistic(FontAwesome icon, int value = 0)
                 Value = value;
             }
         }
+
+        private class TransformEdgeEffectRadius : Transform<float, Container>

```

Comment:
Move this to CompositeDrawable and add to TransformableExtensions + TransformSequence plz.